README.md

```diff
@@ -129,6 +129,7 @@ A config file can manage auditing preferences `audit-ci`. The config file's keys
   "show-not-found": <boolean>, // [Optional] defaults `true`
   "registry": <string>, // [Optional] defaults `undefined`
   "retry-count": <number>, // [Optional] defaults 5
+  "skip-dev": <boolean>, // [Optional] defaults `false`
```

Comment:
There's also the table above that must document `skip-dev`

Reply:
Thanks I missed that. If you want me to change the wording, please let me know.
Greg